src/Athena/Controllers/api/InstitutionCoursesController.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Athena.Core.Repositories;
+using Athena.Core.Models;
+using Athena.Exceptions;
+using System.Net;
+
+namespace Athena.Controllers.api
+{
+    public class InstitutionCoursesController : Controller
+    {
+        private readonly IInstitutionRepository institutions;
+        private readonly ICourseRepository courses;
+
+        public InstitutionCoursesController(IInstitutionRepository institutionRepository,ICourseRepository courseRepository)
+        {
+            institutions = institutionRepository ?? throw new ArgumentNullException(nameof(institutionRepository));
+            courses = courseRepository ?? throw new ArgumentNullException(nameof(courseRepository));
+        }
+
+        [HttpGet("api/v1/institution/{id}/courses")]
```

Comment:
While there is only one action, for consistency hoist `api/v1/institution/{id}` as a `[Route(...)]` attribute on the whole controller.